dask/dask.yaml

```diff
@@ -16,6 +16,7 @@ dataframe:
     metadata-task-size-remote: 1  # Number of files per remote metadata-processing task
   convert-string: null  # Whether to convert string-like data to pyarrow strings
   query-planning: false  # Whether to use dask-expr
+  query-optimization-warning: true  # Warn about future query optimization change
```

Comment:
Could you rename to `query-planning-warning`?

Reply:
Renamed and moved to https://github.com/dask/dask/pull/10925